.github/scale-config.yml

```diff
@@ -45,12 +45,6 @@ runner_types:
     is_ephemeral: false
     max_available: 3120
     os: linux
-  linux.2xlarge.amd:
```

Comment:
Looks like this was added as a test recently: https://github.com/pytorch/test-infra/pull/1097
Maybe worth checking if @malfet still needs it